content/20.treatments.md

```diff
@@ -193,6 +193,8 @@ HCQ and CQ act through the same mechanism of action, but HCQ is regarded to be s
 HCQ/CQ have been reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10/dh8phb; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
 _In vitro_ studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to inhibition of viral entry through elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi:10.1186/1743-422X-2-69].
 In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective in inhibiting viral replication, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
+Coagulopathy and antiphospholipid antibodies were found in patients with Covid-19 [@doi:10.1056/NEJMc2007575]. Antiphospholipid antibodies is central to the diagnosis of the antiphospholipid syndrome. Hydroxychloroquine has been shown to be effective in treating Antiphospholipid Syndrome [@doi:10.1182/blood.V128.22.5023.5023]. 
```

Comment:
```suggestion
Coagulopathy and antiphospholipid antibodies were found in patients with COVID-19 [@doi:10.1056/NEJMc2007575]. 
The presence of antiphospholipid antibodies is central to the diagnosis of the antiphospholipid syndrome. 
Hydroxychloroquine has been shown to be effective in treating Antiphospholipid Syndrome [@doi:10.1182/blood.V128.22.5023.5023]. 
```
Grammatical suggestion -- if you don't want "the presence is" then just swap "is central" to "are central"

Reply:
I'll start with a disclaimer that I have little knowledge of hematology/rheumatology.

With that said, this seems like an interesting mechanism of hydroxychloroquine in COVID-19.

The first sentence refers to a case study of three patients, which I think we need to make clear. Additionally, only one of the three had coagu "A case study of three COVID-19 patients reported presence of antiphospholipid antibodies in all three patients and the addition of coagulopathy and multiple infarcts in one."

I think an additional citation for the last sentence ("Hydroxychloroquine has been shown to be effective in treating Antiphospholipid Syndrome")  should be the following articles: doi:10.1016/j.autrev.2014.01.053 and doi:10.1182/asheducation-2016.1.714

Reply:
Thank you very much for the suggestion! I have revised the text and just committed.